libjulia/backends/evm/EVMCodeTransform.cpp

```diff
@@ -39,8 +39,16 @@ void CodeTransform::operator()(VariableDeclaration const& _varDecl)
 
 	int expectedItems = _varDecl.variables.size();
 	int height = m_assembly.stackHeight();
-	boost::apply_visitor(*this, *_varDecl.value);
-	expectDeposit(expectedItems, height);
+	if (_varDecl.value)
+	{
+		boost::apply_visitor(*this, *_varDecl.value);
+		expectDeposit(expectedItems, height);
+	}
+	else
+	{
+		while (expectedItems--)
```

Comment:
I think it would be cleaner to use a new counting variable. As it currently is, `expectedItems` is zero in the next for loop in one branch, but not in the other.